src/commands.rs

```diff
@@ -2059,17 +2059,36 @@ fn cmd_abandon(
         )
     };
     let mut tx = workspace_command.start_transaction(&transaction_description);
-    for commit in to_abandon {
+    for commit in to_abandon.iter() {
```

Comment:
I typically use `&to_abandon` instead of `to_abandon.iter()`, but I don't know if it's more idiomatic